ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -26,6 +27,44 @@ def argmax(a, axis=None, out=None, keepdims=False):
     return ivy.argmax(a, axis=axis, keepdims=keepdims, out=out, dtype=ivy.int64)
 
 
+# argmin
+@to_ivy_arrays_and_back
+@with_supported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("int16", "int32", "int64")}},
+    "paddle",
+)
```

Comment:
Hi, I set the backend to paddle because after the last update, there was a problem with 'int8' for paddle. This was heavily discussed in the discord at the time as many of us were getting errors. Also, this code does not affect the backend being jax, as it only indicates that for paddle update 2.5.1, the dtypes listed should be used.